Python/initconfig.c

```diff
@@ -2274,8 +2295,11 @@ config_parse_cmdline(PyConfig *config, PyWideStringList *warnoptions,
 
         switch (c) {
         case 0:
-            // Handle long option.
-            assert(longindex == 0); // Only one long option now.
+          // Handle long options.
+          assert(longindex < 2);  // Only two long options for now.
+          switch(longindex) {
+          case 0:
```

Comment:
I think I could actually avoid this nested switch by setting a different `val` in the long option struct I added, let me try that